library/rtsp/src/main/java/com/google/android/exoplayer2/source/rtsp/RtspClient.java

```diff
@@ -161,11 +162,13 @@ public RtspClient(
       PlaybackEventListener playbackEventListener,
       String userAgent,
       Uri uri,
-      boolean debugLoggingEnabled) {
+      boolean debugLoggingEnabled,
```

Comment:
Could you also change the order of these two parameters: making the debugLoggingEnabled the last parameter. Ditto for all other occurrences.